### ISSUE TITLE
Error installing flight according to READ.ME

### ISSUE THREAD

**Issue by jvollme** [rhysnewell/rosella#41]
Hi, I really would like to continue using Rosella in my project, but I have problems with my pre-existing rosella environment (maybe because I upgraded the base python of my conda installation). So I decided to reinstall it in a new environment.
However I keep getting problems with the flight-installation step.
Here is the complete error log from pip: [flight_compilation_error.txt](https://github.com/rhysnewell/rosella/files/13214454/flight_compilation_error.txt)
There seem to be quite a few compilation errors...
What can I do to get a functioning version of Rosella with flight?

here are the infos on my conda environment:
```
     active environment : rosella
    active env location : /home/ww5070/.conda/envs/rosella
            shell level : 2
       user config file : /home/ww5070/.condarc
 populated config files : /home/ww5070/.condarc
          conda version : 23.9.0
    conda-build version : not installed
         python version : 3.11.5.final.0
       virtual packages : __archspec=1=cascadelake
                          __glibc=2.31=0
                          __linux=5.4.0=0
                          __unix=0=0
       base environment : /opt/conda/miniconda  (read only)
      conda av data dir : /opt/conda/miniconda/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/conda/miniconda/pkgs
                          /home/ww5070/.conda/pkgs
       envs directories : /home/ww5070/.conda/envs
                          /opt/conda/miniconda/envs
               platform : linux-64
             user-agent : conda/23.9.0 requests/2.31.0 CPython/3.11.5 Linux/5.4.0-163-generic ubuntu/20.04.6 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

**Comment by rhysnewell** [comment]
Hi,

Those compilation errors look pretty crazy. I imagine you might need to try and create a fresh rosella environment and limit your python version to be <=3.10 if possible. If that doesn't work then I'll have to have a closer look tomorrow

Cheers,

Rhys

**Comment by jvollme** [comment]
apparently, specifying python=3.10 really solved the issue, thanks! 
Probably another specification to keep in mind for the rosella.yml file then I guess... :)

**Comment by rhysnewell** [comment]
Should be fixed in `v0.5.0`
